cns/service/main.go

```diff
@@ -799,14 +799,11 @@ func main() {
 	}
 
 	// Setting the remote ARP MAC address to 12-34-56-78-9a-bc on windows for external traffic if HNS is enabled
-	arpCtx, arpCtxCancel := context.WithTimeout(rootCtx, 30*time.Second)
-	err = platform.SetSdnRemoteArpMacAddress(arpCtx)
+	err = platform.SetSdnRemoteArpMacAddress(rootCtx)
```

Comment:
just curious why drop the timeout?

Reply:
we didn't previously timeout and this is just fully reverting any behavioral change